src/sentry/api/endpoints/release_thresholds/release_threshold_status_index.py

```diff
@@ -69,9 +69,7 @@ class ReleaseThresholdStatusIndexSerializer(serializers.Serializer):
     project = serializers.ListField(
         required=False, allow_empty=True, child=serializers.IntegerField()
     )
-    release_id = serializers.ListField(
-        required=False, allow_empty=True, child=serializers.IntegerField()
-    )
+    release = serializers.ListField(required=False, allow_empty=True, child=serializers.CharField())
```

Comment:
nit: maybe call this release_version or version?

Reply:
I'll update the help text to describe this.

Opting to keep it the shorter 'release' since typically you would reference a release by its version anyways